ci-operator/step-registry/ipi/conf/vsphere/template/ipi-conf-vsphere-template-chain.yaml

```diff
@@ -0,0 +1,15 @@
+chain:
+  as: ipi-conf-vsphere-template
+  steps:
+  - ref: ipi-conf-vsphere-check
+  - ref: ipi-conf-vsphere-vips
```

Comment:
Per `ipi-conf-vsphere-chain.yaml` and `ipi-conf-vsphere-zones-chain.yaml`, sounds like we need to append `ipi-conf-vsphere-vips-vcm` here ?